### PR TITLE
Posasimilaris Intestivirus Submission

### PR DESCRIPTION
**Pull request by DerekTea**

### CODE REVIEW

File: virus/virTemplate.md
Comment (ababaian) [inline virus/virTemplate.md:1]: Please do not modify the Template File in the PR, add your virus as a new file named `virus/p_intestivirus`.

File: virus/virTemplate.md [hunk 1]
```diff
@@ -1,40 +1,106 @@
-# {Q1A Paper Title}
-written by: [yourname](https://github.com/yourname *optional link*)
+### {A Proposed Microbiota Stool Bus as a Mode of Infection for a Novel Posa-like Virus}
+written by: [Derek Tea](https://github.com/DerekTea)
 
-### {Q1B Virus Name}
-*{Q1B, etiology of virus name}*
+### {Posasimilis Intestiviridae}
```
Comment (ababaian) [inline virus/virTemplate.md]: The suffix `viridae` means "Virus Family", so a single virus cannot be a "viridae", change to "Intestivirus" or something along those lines please.

File: virus/virTemplate.md [hunk 2]
```diff
@@ -1,40 +1,106 @@
...
+Virus name: Posasimilis Intestiviridae. I gave my virus this latin binomial name because it is 1) related to the unclassified family of posa-like viruses and 2) it, along with its relatives, are found in stool samples of its associated host. This relates to the story of my virus because I propose a mechanism whereby this virus, and presumably other posa-like viruses, infects the host through the gut microbiota (located in the intestinal tracts). 
 
-[Optional: Tutorial 2 Generative AI Cover-art](img/<virusname>/cover.png)
+Cover art:
+![Posasimilis Intestiviridae](https://github.com/DerekTea/VIRUSxDISCVRY/assets/86479477/622afcb1-19f4-4040-abce-f1cd86152e35)
```
Comment (ababaian) [inline virus/virTemplate.md]: Include this in the PR please with correct file path.

File: virus/virTemplate.md [hunk 3]
```diff
@@ -1,40 +1,106 @@
...
+
+The proposal for a bacterial-mediated entry into the host organism is reflected by the results from another dataset, SRR5830110 (accessible via the SRA database). The sample containing the Posasimilis Intestiviridae hit derives from the fecal community of Tule Elk. Blastx analysis matches the virus with Caudovirales sp. and Candidatus Scalindua sp. as the top hits. The presence of Caudovirales may strengthen the notion that our virus of interest operates in a bacteriophage-like manner. Moreover, Candidatus Scalindua is a genus of marine-associated bacteria first discovered in a waster water treatment plant [Schmid, 2003]. This highlights the capacity for Posasimilis intestinum to infect bacteria, specifically bacteria that may be found in the water supply of wild-life. 
+
+All figures and related information pertaining to dataset runs and Posa can be found in the "MGY441 A3 Virus Seq Runs" csv file.
+[Uploading MGY441 A3 Virus Seq Runs.csv…]()
```
Comment (ababaian) [inline virus/virTemplate.md]: Remove Uploading pending text.

File: virus/virTemplate.md [hunk 4]
```diff
@@ -1,40 +1,106 @@
...
+[MGY441 A3 Virus Seq Runs.csv](https://github.com/DerekTea/VIRUSxDISCVRY/files/13552458/MGY441.A3.Virus.Seq.Runs.csv)
```
Comment (ababaian) [inline virus/virTemplate.md]: Include data file in the `img` path as PR instructions outline.

File: virus/virTemplate.md [hunk 5]
```diff
@@ -1,40 +1,106 @@
...
+
 ### Ecology subsection A...
```
Comment (ababaian) [inline virus/virTemplate.md]: Give this a more informative title fitting to a report, not a "homework assignment"

File: virus/virTemplate.md [hunk 6]
```diff
@@ -1,40 +1,106 @@
...
 
-{Q2 answer ...}
+Facts about Posa-like viruses:
```
Comment (ababaian) [inline virus/virTemplate.md]: Remove fact list please.

File: virus/virTemplate.md [hunk 7]
```diff
@@ -1,40 +1,106 @@
...
+1) Non-structural module located at the 5' end, structural module located at the 3' end, intergenic region in between
+2) The two modules are located in separate genomic segments 
+3) Non-structural module located at the 3' end, structural module located at the 5' end, intergenic region in between
+- The in Picornavirales genome, the RNA is linked to the viral protein at its 5' end [Jiang, 2014]
+- Picornavirales have a poly-A at its 3' end [Jiang, 2014]
+- The OFR of the Picornavirales genome is flanked by a long 5' non-translated region and a short 3' non-translated region [Jiang, 2014]
+- A prominent conserved RNA secondary structure among Picornavirales includes an internal ribosomal entry site (IRES) towards the 5' end [Witwer, 2015]
+- Picornavirales have a capsid protein with icosahedral symmetry [Jiang, 2014]
 
 ### Virus Genome {Q3}
```
Comment (ababaian) [inline virus/virTemplate.md]: Convert this to `.png` and include it embedded as an image. The figure legend should be written into the markdown file given below the figure.

File: virus/virTemplate.md [hunk 8]
```diff
@@ -1,40 +1,106 @@
...
 
-{Insert your Virus Genome Map Figure and figure legend here}
+[MGY441 A3 Viral Genome.pdf](https://github.com/DerekTea/VIRUSxDISCVRY/files/13552439/MGY441.A3.Viral.Genome.pdf)
```
Comment (ababaian) [inline virus/virTemplate.md]: I'd also recommend moving up your Virus Genome map to the top of the Results section. This makes a more natural fit there prior to the ecology.

File: virus/virTemplate.md [hunk 9]
```diff
@@ -1,40 +1,106 @@
...
 
 ### Other (bonus) sections
```
Comment (ababaian) [inline virus/virTemplate.md]: Better section title please.

File: virus/virTemplate.md [hunk 10]
```diff
@@ -1,40 +1,106 @@
...
 
-{Q4: Create sub-sections for any bonus or other cool findings you've made
-related to your virus}
+**Predicted 3D Structure**
+![Phyre2 3D Structure Motifs](https://github.com/DerekTea/VIRUSxDISCVRY/assets/86479477/5ffb69be-11b9-469f-aade-7a391094007b)
```
Comment (ababaian) [inline virus/virTemplate.md]: Embed images into PR. Please add descriptive figure legend for each of these.

**Comment by DerekTea** [comment]
Posasimilaris Intestivirus A3 changes

**Review by DerekTea — COMMENTED** [review]
Posasimilaris Intestivirus changes

**Comment by DerekTea** [comment]
Hi professor! I didn't have the "img" file in my virus fork despite forking the entire repository for some reason... i created by own folder named img i hope that works. Thanks!

File: virus/img/p_intestivirus/Cobalt Phylogenetic Tree.png
Comment (ababaian) [inline virus/img/p_intestivirus/Cobalt Phylogenetic Tree.png:1]: please don't use spaces in your file names.

File: virus/virTemplate.md
Comment (ababaian) [inline virus/virTemplate.md:1]: Please don't delete thetempalte file in your PR.

File: virus/p_intestivirus [hunk 11]
```diff
@@ -0,0 +1,84 @@
+### {A Proposed Microbiota Stool Bus as a Mode of Infection for a Novel Posa-like Virus}
+written by: [Derek Tea](https://github.com/DerekTea)
+
+[Cover art Intestivirus](virus/img/p_intestivirus/cover_art_intestivirus.jfif)
```
Comment (ababaian) [inline virus/p_intestivirus]: Image paths should be `(img/p_intesti...)` The file is already in the `virus/` path and so you're telling it to look for the image in `virus/virus/img/` which does not exist.

File: virus/p_intestivirus [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+
+Related information pertaining to dataset runs can be found in the "MGY441 A3 Virus Seq Runs" csv file.
+
+[MGY441 A3 Virus Seq Runs.csv](virus/img/p_intestivirus/MGY441 A3 Virus Seq Runs.csv)
+Table 1: A 'Top Hits' Analysis for Each P.Intestivirus Accession Code Hit. 
+
+# Viral Genome 
+
+[MGY441 A3 Viral Genome.png](virus/img/p_intestivirus/MGY441 A3 Viral Genome.png)
+Figure 1. A) Proposed Genomic Architecture of the Posasimilaris Intestivirus. Conserved sequences were identified with Blastp and aligned with the coding amino acid sequence of the virus. Distance representation between conserved regions are not to scale. The presence of the 5’NTR and 3’NTR is based on the conserved 5’NTR and 3’NTR domains found in Picornavirales [Jiang, 2014]. Size and length is not to scale.  B) Predicted RNA Secondary Structure of Posasimilis Intestiviridae. The prediction was performed with the RNAfold tool, utilizing minimal free energy predictions only.
```
Comment (ababaian) [inline virus/p_intestivirus]: For all the Figure Legends, add a `>` in front of the line to make it "Quote Text". This will look nicer.

**Review by ababaian — CHANGES_REQUESTED** [review]

**Review by ababaian — APPROVED** [review]

**Pull request merged**